mbedtls/Cargo.toml

```diff
@@ -57,7 +57,7 @@ cc = "1.0"
 default = ["std", "aesni", "time", "padlock"]
 std = ["mbedtls-sys-auto/std", "serde/std", "yasna"]
 debug = ["mbedtls-sys-auto/debug"]
-no_std_deps = ["core_io", "spin"]
+no_std_deps = ["core_io", "spin", "serde/alloc"]
```

Comment:
Could you explain why is this feature needed?

Also, what about `core_io`? Are you using a patched version (I see https://github.com/jethrogb/rust-core_io/pull/36)?